flamedisx/inference.py

```diff
@@ -220,7 +261,10 @@ def minimize(self):
                 pd.DataFrame(dict(y=[h['y'] for h in self._history])),
                 pd.DataFrame([h['grad'] for h in self._history],
                              columns=[k + '_grad'
-                                      for k in self.arg_names])
+                                      for k in self.arg_names]),
+                pd.DataFrame([h['scaled_grad'] for h in self._history],
+                             columns=[k + '_scaled_grad'
+                                      for k in self.arg_names]),
             ], axis=1)
         result, llval = self.parse_result(result)
```

Comment:
Rather than undoing the scaling in each minimizer implementation, could we do it here?